controllers/constants/runtime-metrics.go

```diff
@@ -270,6 +270,68 @@ const (
 						"query":  "sum(container_memory_working_set_bytes{namespace='${NAMESPACE}', pod=~'${MODEL_NAME}-predictor-.*'})/sum(kube_pod_resource_limit{resource='memory', pod=~'${MODEL_NAME}-predictor-.*', namespace='${NAMESPACE}'})"
 					}
 				]
+			},
+			{
+				"title": "GPU cache usage over time",
+				"type": "KV_CACHE",
+				"queries": [
+					{
+						"title": "GPU cache usage over time",
+						"query":  "TODO"
```

Comment:
will this be added for now?

Reply:
Yes, we're looking for some clarifications about this graph. But this will be included in this PR ASAP.